linkerd/proxy/transport/src/dual_bind.rs

```diff
@@ -0,0 +1,63 @@
+use crate::{
```

Comment:
nit: Would you mind moving this into `linkerd/proxy/transport/src/listen/dual.rs` so that it's coped under the listen module?